spec/spec_helper.rb

```diff
@@ -105,3 +113,5 @@
   Kernel.srand config.seed
 =end
 end
+
+require "rails-patterns"
```

Comment:
This looks awkward, as we do not know the reason behind it. Can you please move `Rails` mocking only to spec file that uses it ?